--- old.dts	2022-02-13 15:22:27.993332891 +0100
+++ new.dts	2022-02-13 15:25:41.566666176 +0100
@@ -2066,21 +2066,21 @@
 
 		opp-792000000 {
 			opp-hz = <0x00 0x2f34f600>;
-			opp-microvolt = <0xf4240>;
+			opp-microvolt = <0xc96a8>;
 			clock-latency-ns = <0x249f0>;
 			opp-supported-hw = <0x0f 0x0f>;
 		};
 
 		opp-996000000 {
 			opp-hz = <0x00 0x3b5dc100>;
-			opp-microvolt = <0x10c8e0>;
+			opp-microvolt = <0xe1d48>;
 			clock-latency-ns = <0x249f0>;
 			opp-supported-hw = <0x0c 0x0f>;
 		};
 
 		opp-1200000000 {
 			opp-hz = <0x00 0x47868c00>;
-			opp-microvolt = <0x12b128>;
+			opp-microvolt = <0x100590>;
 			clock-latency-ns = <0x249f0>;
 			opp-supported-hw = <0x08 0x0f>;
 		};
